kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -288,6 +290,7 @@ showRule
     -> m ()
 showRule configNode = do
     Strategy.ExecutionGraph { graph } <- Lens.use lensGraph
+    ReplState { axioms } <- get
```

Comment:
You could probably combine this line with the above into something like

`ReplState { axioms, graph = Strategy.ExecutionGraph { graph } } <- get`

... I think?